Doc/tutorial/introduction.rst

```diff
@@ -52,8 +52,8 @@ Numbers
 
 The interpreter acts as a simple calculator: you can type an expression at it
 and it will write the value.  Expression syntax is straightforward: the
-operators ``+``, ``-``, ``*`` and ``/`` work just like in most other languages
-(for example, Pascal or C); parentheses (``()``) can be used for grouping.
+operators ``+``, ``-``, ``*`` and ``/`` work similarly to how you perform
+arithmetic operations in mathematics; parentheses (``()``) can be used for grouping.
```

Comment:
```suggestion
operators ``+``, ``-``, ``*`` and ``/`` can be used to perform
arithmetic; parentheses (``()``) can be used for grouping.
```
more concise, slightly more informal, no need to use "similarly" to plant doubts in beginners' minds